pkg/config/config.go

```diff
@@ -47,3 +49,10 @@ var (
 		"StatefulSet",
 	}
 )
+
+//LogDefaults sets default glog flags
+func LogDefaultFlags() {
+	flag.Set("logtostderr", "true")
+	flag.Set("stderrthreshold", "WARNING")
+	flag.Set("v", "2")
```

Comment:
Please define verbosity levels in config package:
0 - LogInfo
1 - LogVerbose
2 - LogTrace
These levels should be used for logging Info messages at different level. If you have other ideas about the level of verbosity, please share them.

Also, add optional parameter to kyverno that will define log level (default should be 0).

Reply:
create a new issue #129 129